app/assets/javascripts/discourse/app/controllers/full-page-search.js

```diff
@@ -252,15 +254,15 @@ export default Controller.extend({
     return I18n.t("search.result_count", { count, plus, term });
   },
 
-  @observes("model.[posts,categories,tags,users].length")
+  @observes("model.[posts,categories,tags,users].length", "searchResultPosts")
```

Comment:
I know this was already there but… does this dependent key syntax even work? 😄 cc: @davidtaylorhq 

It should be `{}`?

```js
  @observes("model.{posts,categories,tags,users}.length", "searchResultPosts")
```

(see https://guides.emberjs.com/v3.12.0/object-model/computed-properties/ and https://guides.emberjs.com/v3.12.0/object-model/computed-properties-and-aggregate-data/ for the old docs)